components/http/sources/http-webhook-requests/http-webhook-requests.js

```diff
@@ -0,0 +1,50 @@
+const httpApp = require("../../http.app");
+
+// Core HTTP component
+// Returns a 200 OK response, emits the HTTP payload as an event
+module.exports = {
+  key: "http-new-requests-payload-only",
```

Comment:
The other source in this PR is more configurable than this one. Why don't we add a prop there to allow switching between body or raw body, and delete this source to avoid code duplication?